<launch>
    <node name="roboteq_can" pkg="roboteq_node" type="roboteq_node" output="screen" >
        <param name="mode" value="can" type="str" />
        <param name="device" value="/dev/ttyUSB0" type="str" />
        <param name="left" value="1" type="str" />
        <param name="right" value="2" type="str" />
    </node>
</launch>
